vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -206,53 +205,38 @@ def save_network_inventory(guest_device, hash)
   def save_networks_inventory(hardware, hashes, mode = :refresh)
     return if hashes.nil?
 
-    deletes = hardware.networks(true).dup
-
     case mode
     when :refresh
+      deletes = hardware.networks(true).dup
       # Remove networks that were already saved via guest devices
       saved_hashes, new_hashes = hashes.partition { |h| h[:id] }
       saved_hashes.each { |h| deletes.delete_if { |d| d.id == h[:id] } } unless deletes.empty? || saved_hashes.empty?
 
       self.save_inventory_multi(:networks, Network, hardware, new_hashes, deletes, :ipaddress, nil, :guest_device)
     when :scan
-      self.save_inventory_multi(:networks, Network, hardware, hashes, deletes, [:description, :guid])
+      save_inventory_multi(:networks, Network, hardware, hashes, true, [:description, :guid])
```

Comment:
Oh nvm...I didn't see the new commit

Can you change the PR title, since it isn't accurate anymore?